config/routes.rb

```diff
@@ -4,9 +4,14 @@
   # Defines the root path route ("/")
   # root "articles#index"
   root "users#index"
+
+  get '/posts/new', to: 'posts#new', as: 'create_post'
+  post '/posts/new', to: 'posts#create'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'post'
+  post '/users/:user_id/posts/:id/', to: 'posts#create_comment'
+  post '/users/:user_id/posts/:id/likes', to: 'posts#create_like', as: 'like'
+
```

Comment:
- [ ] Kindly ensure that there is a pair of routes in the `config/routes.rb` file which corresponds with the `new` and `create`, actions in the comments controller. Viewing these routes in the command line should be similar to:
```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new